main program/main.cpp

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (C) 2022 Carry Assistant Trolley (C.A.T) Technologies
+ *  
+ * Jia Wei Tan, February 26, 2022.
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02111, USA.
+ */
+
+#include <iostream>
+
+#include <cstdlib>
+
+using namespace std;
+
+int main() {
+    /*initialize variables
+     * a: RSSI robot--phone
+     * b: RSSI beacon--robot
+     * c: RSSI beacon--phone
+     */
+    int a, b, c;
+    float angle_old = 0;
+    float angle_new = 0;
+    int followDistance = 1; //unit: metre
+    int angleMove = 15; //angle to accumulate before moving
+    int * rotation;
```

Comment:
Angle correction function when written would return a pointer? As it says " int * rotation ... "

Is that correct ? @jwtan5